src/pages/_index/_components/what-is-wasmcloud/index.tsx

```diff
@@ -0,0 +1,194 @@
+import styles from './what-is-wasmcloud.module.css';
+
+type Props = {};
+
+const INTRO_CONTENT: {
+  tag: React.ReactNode;
+  heading: React.ReactNode;
+  description: React.ReactNode;
+} = {
+  tag: 'What is wasmCloud?',
+  heading: 'Fast-track the development, deployment, and management of distributed applications',
+  description: (
+    <p>
+      wasmCloud is an open source project from the Cloud Native Computing Foundation (CNCF) that
+      enables teams to build polyglot applications composed of reusable code and run
+      them—resiliently and efficiently—across any Kubernetes cluster, cloud, datacenter, or edge.
+    </p>
+  ),
+};
+
+const SWITCHER_CONTENT: Array<{
+  id: string;
+  title: React.ReactNode;
+  image: string;
+  features: Array<{
+    title: React.ReactNode;
+    description: React.ReactNode;
+    link?: string;
+    linkText?: React.ReactNode;
+  }>;
+}> = [
+  {
+    id: 'build',
+    image: '/img/pages/home/what-is-wasmcloud/build.svg',
+    title: 'Build',
+    features: [
+      {
+        title: 'Integrate with your existing ecosystem',
+        description: (
+          <>
+            wasmCloud has first-tier support for Kubernetes, VS Code, ArgoCD, Backstage, Chainguard,
+            and more.
+          </>
+        ),
+      },
+      {
+        title: 'Build Reusable Components',
+        description: (
+          <>
+            Using reusable components means that platform teams can build features that can be
+            leveraged across your entire organization.
+          </>
+        ),
+      },
+      {
+        title: 'Bring-your-own language(s) for polyglot apps',
+        description: (
+          <>
+            Build and run applications from components written in C, C#, C++, Rust, Python, Go, C#,
+            TypeScript and more.
+          </>
+        ),
+      },
+    ],
+  },
+  {
+    id: 'compose',
+    image: '/img/pages/home/what-is-wasmcloud/compose.svg',
+    title: 'Compose',
+    features: [
+      {
+        title: 'Combine components into applications',
+        description: (
+          <>
+            Create applications by connecting WebAssembly components, first-party platform
+            capabilities, or your own extensions.
+          </>
+        ),
+      },
+      {
+        title: 'Elastically extensible',
+        description: (
+          <>
+            Easily extend the wasmCloud platform at runtime with secure, plugin-based custom
+            capabilities—including your own custom interfaces.
+          </>
+        ),
+      },
+      {
+        title: 'Portable and pluggable',
+        description: (
+          <>
+            Universal interface abstractions built on WebAssembly Interface Types (WIT) maximize
+            application portability across dev, QA, test, and prod.
+          </>
+        ),
+      },
+    ],
+  },
+  {
+    id: 'run',
+    image: '/img/pages/home/what-is-wasmcloud/run.svg',
```

Comment:
This image looks like the `k8s_region`'s N got pushed to the other side 
<img width="514" alt="image" src="https://github.com/user-attachments/assets/2db52e19-8873-4406-b64a-d9b9a30a8989">